src/test/java/org/isf/pregtreattype/test/Tests.java

```diff
@@ -116,6 +121,141 @@ public void testIoDeletePregnantTreatmentType() throws Exception {
 		assertThat(result).isFalse();
 	}
 
+	@Test
+	public void testMgrGetPregnantTreatmentType() throws Exception {
+		String code = _setupTestPregnantTreatmentType(false);
+		PregnantTreatmentType foundPregnantTreatmentType = pregnantTreatmentTypeIoOperationRepository.findOne(code);
+		ArrayList<PregnantTreatmentType> pregnantTreatmentTypes = pregnantTreatmentTypeBrowserManager.getPregnantTreatmentType();
```

Comment:
if you setup more PregnantTreatmentType elements, then you could assert that the method under tests getPregnantTreatmentType() returns a list of the expected size

Reply:
I created two more instances and did a `hasSize(3)` assertion.  Thanks